test/e2e/qat/qatplugin_dpdk.go

```diff
@@ -93,12 +101,14 @@ func describeQatDpdkPlugin() {
 		}
 	})
 
-	ginkgo.Context("When QAT Gen4 resources are available", func() {
+	ginkgo.Context("When QAT Gen4 resources are available with crypto (cy) services enabled", func() {
+		// This BeforeEach runs even before the JustBeforeEach above.
 		ginkgo.BeforeEach(func() {
-			ginkgo.By("checking if the resource is allocatable")
-			if err := utils.WaitForNodesWithResource(f.ClientSet, "qat.intel.com/cy", 30*time.Second); err != nil {
-				framework.Failf("unable to wait for nodes to have positive allocatable resource: %v", err)
-			}
+			ginkgo.By("creating a configMap before plugin gets deployed")
+			e2ekubectl.RunKubectlOrDie(f.Namespace.Name, "create", "configmap", "--from-literal", "qat.conf=ServicesEnabled=sym;asym", "qat-config")
+
+			ginkgo.By("setting resourceName for cy services")
+			resourceName = "qat.intel.com/cy"
```

Comment:
is this enough to guarantee that the provisioning works and that there are no dc resources left?

Reply:
maybe discussed enough in 1:1 chat.!
after this pr is merged, I will add a test case for dc with compress test app.
Then, it would have more coverage.